navigation2/package.xml

```diff
@@ -15,20 +15,22 @@
   <!-- nav2_bringup doesn't belong for recursive dependencies -->
   <exec_depend>nav2_amcl</exec_depend>
   <exec_depend>nav2_bt_navigator</exec_depend>
+  <exec_depend>nav2_controller</exec_depend>
```

Comment:
This doesn't appear to exist. I think you mean `nav2_dwb_controller`.

Reply:
@SteveMacenski It's over here: #662. There are a series of PRs with some dependencies. After the first few PRs are integrated into the lifecycle branch. Subsequent PRs should compile. It will take all of the PRs to get everything to build. At that point, we'll test out the lifecycle branch before considering a merge to master.

Reply:
Maybe we should rename it to `nav2_lifecycle_controller`? It is kind of easy to confuse with `nav2_dwb_controller` as is.

Reply:
@crdelsey Yes, I can rename. Mohammad has also suggested renaming. Perhaps nav2_lifecycle_manager to avoid potential confusion with lower-level controllers. The new name would be consistent with "managed nodes." Shall I rename with this PR or do it as a separate step after integrating the series of PRs?

Reply:
Well, addressing review comments in the same PR would help track what's done and what's not and help prevent missing an important fix.

Reply:
@crdelsey Ok, I'll rename it here. What name to use has come up on another thread. Matt has proposed generalizing the name to nav2_system_controller.

Reply:
Ok, updated in this PR. Others will follow...

Reply:
@SteveMacenski OK to merge?